docs/sources/developers/angular_deprecation/_index.md

```diff
@@ -14,7 +14,17 @@ weight: 500
 
 # Angular support deprecation
 
-Angular plugin support is deprecated and will be removed in a future release. There are still many community plugins that rely on Grafana's Angular plugin support to work. The same is true for many internal (private) plugins that have been developed by Grafana users over the years. Grafana version 9 has a server configuration option that is global to the entire instance and controls whether Angular plugin support is available or not. By default, Angular support is still enabled, but that will change soon once we complete the migration of all Angular code in the core product.
+Angular plugin support is deprecated and will be removed in a future release. There are legacy Core Grafana visualizations and external plugins that rely on Grafana's Angular plugin support to work. The same is true for many internal (private) plugins that have been developed by Grafana users over the years. From Grafana v9 and onwards, there is a [server configuration option](<(https://github.com/grafana/grafana/blob/d61bcdf4ca5e69489e0067c56fbe7f0bfdf84ee4/conf/defaults.ini#L362)>) that is global to the entire instance and controls whether Angular plugin support is available or not. By default, Angular support is still enabled, but that will change soon once we complete the migration of all Angular code in the core product.
+
+Warning messages are displayed if a dashboard depends on an a panel visualization or data source which requires AngularJS as shown in the following video:
+
+{{< youtube id="XlEVs6g8dC8" >}}
+
+To avoid disruption:
+
+- Ensure that you are running the latest version of plugins by following this guide on [updating]({{< relref "../../administration/plugin-management/#update-a-plugin" >}}). Many panels and data sources have migrated from AngularJS.
+- If you are using legacy Core Grafana visualizations such as Graph or Table-old, migrate to their replacements using the provided [automatic migrations]({{< relref "./angular-plugins/#automatic-migration-of-plugins" >}}).
+- Review the [list of current Angular plugins]({{< relref "./angular-plugins/" >}}) to discover which Core and external plugins are impacted, and whether an update or alternative is required.
```

Comment:
- Links directly to the table listing AngularJS plugins
- Reorders the list to link to the table first
- Reword to provide lead in for links to tasks
- Uses semantic line breaks
- Removes capitalization of "Core" because I'm not sure that it's a proper noun

 ```suggestion
- Review the [list of current Angular plugins]({{< relref "./angular-plugins#angularjs-based-plugins" >}}) to discover which core and external plugins use Angular, and whether you need to update or use an alternative.
- Ensure that you are running the latest version of plugins.
  Many panels and data sources have migrated from AngularJS.
  To update a plugin, refer to [Update a plugin]({{< relref "../../administration/plugin-management/#update-a-plugin" >}}).
- If you are using legacy Grafana visualizations such as Graph or Table (old), migrate to their replacements.
  To migrate your visualizations automatically, refer to [Automatic migration of plugins]({{< relref "./angular-plugins#automatic-migration-of-plugins" >}}).
```